screensaver/screensaver.py

```diff
@@ -120,7 +122,7 @@ def upload_new_screensaver_photos(photo_dirs_list: list) -> None:
                            # exclude files:
                            "--exclude", "*",
                            # from:
-                           f"{INPUT_PATH}'{random_dir_choice}'",
+                           f"{INPUT_PATH}{random_dir_choice}/",
```

Comment:
Again, we need to write tests for this function really, because from what I remember, I added the single quotes to handle white spaces in the directory name. Have you tested this works with different directory names, especially one with white spaces? What's the behaviour for other cases?

Reply:
Yes, this probably needs a test in the future. 
The problem of whitespace is taken care of by the added rsync-flag '--protect-args, -s', which transmits arguments to the server side with no space-splitting (& wildcard chars only) according to the documentation. Therefore this should work with all directory names.

Reply:
👍  Great! Neat use of the `-s` flag, didn't know this did this!